src/components/subscriptions/LicenseActions.jsx

```diff
@@ -25,6 +25,7 @@ export default function LicenseAction({ user }) {
 
   const hasNoRevocationsRemaining = useHasNoRevocationsRemaining(details);
   const noActionsAvailable = [{ key: 'no-actions-here', text: '-' }];
+  const { user } = props;
```

Comment:
nit: curious the reason for moving the props destructing for `user`  from the function argument to here?

Reply:
I think this was an attempt to see if moving it would get rid of that buggy eslint error.